src/ServerAdministration/WindowsService/WheelMUDService.cs

```diff
@@ -8,60 +8,37 @@
 // </summary>
 //-----------------------------------------------------------------------------
 
+using Topshelf;
```

Comment:
(I prefer if usings are sorted just before checkin.)  This raises the question about how we want to handle code styling.  Since debates trying to land on all individual aspects of code styling are generally never-ending, it's often best to head that off at the pass - WheelMUD adopted the strategy of "pick an enforcement tool, and migrate towards compliance; each person will probably disagree with a few rules but the benefits of consistency will be worth it in the end."  Basically having a tool arbitrate instead of having arguments.  We chose StyleCop.  I'm wondering how you feel about it.